extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -460,6 +460,20 @@ export class AmpAnalytics extends AMP.BaseElement {
       }
     }
 
+    // Do NOT allow inline or remote config to use 'transport: iframe'
+    if (inlineConfig['transport'] && inlineConfig['transport']['iframe']) {
```

Comment:
Might consider adding && !getMode().localDev

Reply:
Good idea. Add `!getMode().localDev` for local development.